spec/controllers/scim_rails/scim_users_controller_spec.rb

```diff
@@ -569,6 +569,23 @@
         response_body = JSON.parse(response.body)
         expect(response_body.dig("schemas", 0)).to eq "urn:ietf:params:scim:api:messages:2.0:Error"
       end
+
+      it "throws an error for non object values" do
+        patch :patch_update, params: {
+          id: 1,
+          Operations: [
+            {
+              op: "replace",
+              path: "displayName",
+              value: "Francis"
```

Comment:
This is the format of the bad operations we were seeing in production. They had paths with string values.